connectors/src/connectors/slack/temporal/activities.ts

```diff
@@ -159,6 +175,7 @@ export async function syncNonThreaded(
       messages.push(message);
     }
   }
+  messages.reverse();
```

Comment:
This seems to impact the initial sync. Was it wrong before?

Reply:
yes it was. Messages were reversed for non threaded messages.
This is because we added the time bounds, so the order is automatically reversed by the API :/